File: leetcode/reverse-integer/Driver.java
public class Driver {
  public static void main(String[] args) {
    Solution s = new Solution();

    for (int i = -20; i <= 20; i++) {
      int reversed = s.reverse(i);
      System.out.printf("%d %d\n", i, reversed);
    }
  }
}

File: leetcode/first-missing-positive/Driver.scala
object Driver {
  def main(a: Array[String]) {
    val s = new Solution()
    println(s.firstMissingPositive(Array(1, 1)))
    println(s.firstMissingPositive(Array(1, 2, 0)))
    println(s.firstMissingPositive(Array(-1, 2, 0, 3)))
    println(s.firstMissingPositive(Array(3, 4, -1, 1)))
    println(s.firstMissingPositive(Array(1000, -1)))
    println(s.firstMissingPositive(Array(1000, 1)))
  }
}
